src/org/sparkboard/config.edn

```diff
@@ -9,4 +9,8 @@
  {:server/host #profile {:dev #join["http://localhost:" #ref [:common :server/port]]
                          :staging "https://staging.sparkboard.dev"
                          :prod "https://prod.sparkboard.dev"}
-  :server/port #or [#env PORT 8000]}}
\ No newline at end of file
+  :server/port #or [#env PORT 8000]
+  :firebase/config
+  #profile {:dev "{\"apiKey\":\"AIzaSyBS2FVATX9zvytieEgK-1sPGmI3DU5JUhY\",\"authDomain\":\"sparkboard-dev-482f0.firebaseapp.com\",\"databaseURL\":\"https://sparkboard-dev-482f0.firebaseio.com\",\"projectId\":\"sparkboard-dev\",\"storageBucket\":\"sparkboard-dev.appspot.com\",\"messagingSenderId\":\"94039713434\",\"appId\":\"1:94039713434:web:a448eb6375ac640f\"}"
+            :staging "{\"apiKey\":\"AIzaSyAhvfCsdeFca1l22fhVR55S9zQxIZNWT8c\",\"authDomain\":\"sparkboard-staging.firebaseapp.com\",\"databaseURL\":\"https://sparkboard-staging.firebaseio.com\",\"projectId\":\"sparkboard-staging\",\"storageBucket\":\"sparkboard-staging.appspot.com\",\"messagingSenderId\":\"146437767784\",\"appId\":\"1:146437767784:web:24ab716e60bc95219a71dd\"}"
+            :prod "{\"apiKey\":\"AIzaSyCTziMxCUG2nBPhg9JNqZ9nTEs7OYxVfuI\",\"authDomain\":\"sparkboard-002.firebaseapp.com\",\"databaseURL\":\"https://sparkboard-002.firebaseio.com\",\"projectId\":\"sparkboard-002\",\"storageBucket\":\"sparkboard-002.appspot.com\",\"messagingSenderId\":\"476510914514\",\"appId\":\"1:476510914514:web:fc0db0bfdf6515d2d4aa0b\"}"}}}
```

Comment:
Is this OK?

Reply:
yeah - these are all public configuration points that would normally go in an `index.html` file. However, maybe I'll move them into an environment variable anyway. I get warnings in my email when I deploy this from some 3rd-party service that thinks the apiKey should be private.